mmpose/models/detectors/posewarper.py

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+
+import numpy as np
+import torch
+
+from ..builder import POSENETS
+from .top_down import TopDown
+
+try:
+    from mmcv.runner import auto_fp16
+except ImportError:
+    warnings.warn('auto_fp16 from mmpose will be deprecated from v0.15.0'
+                  'Please install mmcv>=1.1.4')
+    from mmpose.core import auto_fp16
+
+
+@POSENETS.register_module()
+class PoseWarper(TopDown):
+    """Top-down pose detectors for video inputs.
```

Comment:
If this class is just for the pose warper algorithm, instead of general video pose estimation, the description here should be more specific, and include paper reference.

Reply:
OK, I will change it.